tox.ini

```diff
@@ -19,6 +19,12 @@ commands =
     coverage combine
     coverage report -m
 
+[testenv:idna]
```

Comment:
This should not be a stand-alone `idna` testenv, it should be a modified on the existing pyXX testenvs. See pyOpenSSL's for an example of that syntax